### ISSUE TITLE
Module resolution root path setting

### ISSUE THREAD

**Issue by benmosher** [benmosher/eslint-plugin-import#18]
Need a root path, a la webpack's `resolve.root`. i.e., if `.eslintrc` has something like
```yaml
---
settings:
  resolve.root: 'src'
```

...then the resolver for `resolve.js` should look in `path.join(process.cwd(), 'src')` as an additional base path.

Extra credit: allow specification of a Node-compatible JS/JSON file with a path (i.e. './webpack.config.js:resolve.root').

May want to spec out some environment-based defaults (webpack, browserify, requirejs, explicitly check for node, etc.)

Need to investigate proper setting key conventions (i.e. `import/resolveRoot`?) within ESLint plugins.
